src/store/actions/actions-model.ts

```diff
@@ -70,32 +70,76 @@ export const setCertifyState: ActionIF = ({ commit }, certifyState: CertifyIF):
   commit('mutateCertifyState', certifyState)
 }
 
-export const setCertifyStateValidity: ActionIF = ({ commit }, validity): void => {
-  commit('mutateCertifyStateValidity', validity)
+export const setAlterationCertifyStateValidity: ActionIF = ({ commit }, validity): void => {
```

Comment:
A down side of this is now the state is larger and there is lots of code repetition because each of the filings need their own mutations and actions instead of having it more generalized

Reply:
If we go this route then I'd like to see it generalized again ... possibly using the list of components for each filing in their resource file?

Reply:
Okay that makes sense. Sort of like using any interface but then having the code generalized in the store?